File: gpt-4o/Pyhon/2-3.py
def file_copy():
    input_file_name = input("Enter the name of the input file: ")
    output_file_name = input("Enter the name of the output file: ")

    try:
        with open(input_file_name, 'r') as input_file:
            content = input_file.read()

        with open(output_file_name, 'w') as output_file:
            output_file.write(content)

        print("File content copied successfully.")
    except FileNotFoundError:
        print("Error: The input file was not found.")
    except IOError as e:
        print(f"An error occurred: {e}")

file_copy()

File: Gemini Advanced/C++/2-3.c
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main() {
    string inputFilename, outputFilename;

    cout << "Enter input file name: ";
    cin >> inputFilename;

    cout << "Enter output file name: ";
    cin >> outputFilename;

    ifstream inputFile(inputFilename);
    ofstream outputFile(outputFilename);

    if (!inputFile.is_open()) {
        cout << "Error opening input file!" << endl;
        return 1;
    }

    if (!outputFile.is_open()) {
        cout << "Error opening output file!" << endl;
        return 1;
    }

    string line;
    while (getline(inputFile, line)) {
        outputFile << line << endl;
    }

    inputFile.close();
    outputFile.close();

    cout << "File copied successfully!" << endl;

    return 0;
}

